crates/katana-core/src/accounts.rs

```diff
@@ -78,6 +79,37 @@ impl Account {
     }
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct AccountInformation {
```

Comment:
A bit redundant to have both `Acccount` and `AccountInformation` types imo. Reusing `Account` might be better.

I think we can just remove `contract_class` field from `Account` and pass it as arg when declaring the accounts [here](https://github.com/dojoengine/dojo/blob/0fd240712c2761a438b57fba788f92aca7746fec/crates/katana-core/src/accounts.rs#L54).

Reply:
Ok I have removed `contract_class` field from `Account` and reuse `Account`.